src/overlays/actors/ovl_En_Wf/z_en_wf.c

```diff
@@ -412,7 +412,7 @@ s32 func_80990948(GlobalContext* globalCtx, EnWf* this, s16 arg2) {
     }
 
     if (func_800BE184(globalCtx, &this->actor, 100.0f, 10000, 12000, this->actor.shape.rot.y) &&
-        ((player->swordAnimation == 0x11) || ((globalCtx->gameplayFrames % 2) != 0))) {
+        ((player->meleeWeaponAnimation == 0x11) || ((globalCtx->gameplayFrames % 2) != 0))) {
```

Comment:
These need to be changed to dec at some point, I suppose. Or actually make an enum once we've documented them, which will be easy enough for once.

Reply:
So, you want me to make an enum now? Or should we wait instead?

Reply:
I can fire up KZ later, probably won't be hard to find the array in Player.

Reply:
I fired up KZ, documentation complete: https://docs.google.com/spreadsheets/d/1PPZof5fC0ujw9OupEezdE0XOjCqbGaDUgY24mZ597Xs/edit?usp=sharing
Not saying the names are any good, though.